docs/API_v1/transactions.md

```diff
@@ -6,8 +6,40 @@ Transactions API
 
 
 ### Payment transaction  [POST] 
-//TODO: How to make a transaction and sign 
 
++ Data Body consist of 3 parts, ; T, H, B
```

Comment:
Would be good to add an example here, with dummy values.

Reply:
@cmcm2222 This dummy data should be fixed. the `\"payment\"` is wrong, it should be `"type": "payment"`.